cmake/Modules/FindDPCPP.cmake

```diff
@@ -38,7 +38,7 @@ set(DPCPP_USER_FLAGS "" CACHE STRING
     "Additional user-specified compiler flags for DPC++")
 
 get_filename_component(DPCPP_BIN_DIR ${CMAKE_CXX_COMPILER} DIRECTORY)
-find_library(DPCPP_LIB_DIR NAMES sycl PATHS "${DPCPP_BIN_DIR}/../lib")
+find_library(DPCPP_LIB_DIR NAMES sycl sycl6 PATHS "${DPCPP_BIN_DIR}/../lib")
```

Comment:
May be we should have a comment here to say that this was the last sycl/dpcpp version for which windows support was tested on.

Reply:
It might worth considering backward compatibility as well, if and when needed